web/src/main/java/com/bazaarvoice/emodb/web/resources/blob/BlobStoreResource1.java

```diff
@@ -391,7 +386,6 @@ private String safeResponseContentType(String metadataContentType) {
     public SuccessResponse put(@PathParam("table") String table,
                                @PathParam("blobId") String blobId,
                                InputStream in,
-                               @QueryParam("ttl") SecondsParam ttlParam,
```

Comment:
By removing this, we will still continue to accept TTL's as parameters, we just won't be able to do anything with it. In other words, we will fail to apply the TTL silently. It's probably better to leave this value in, and throw a 400 if it is specified.

Reply:
makes sense, will throw `IllegalArgumentExecption`